mlfinlab/bet_sizing/bet_sizing.py

```diff
@@ -6,29 +6,43 @@
 
 
 # imports
-import numpy as np
-import pandas as pd
-from scipy.stats import norm
+from mlfinlab.bet_sizing.ef3m import M2N
+from mlfinlab.bet_sizing.ch10_snippets import (get_signal, avg_active_signals,

```

Comment:
Remove ()

Reply:
The brackets are used as described in [PEP 328](https://www.python.org/dev/peps/pep-0328/) "Rationale for Parentheses" on how to handle a long string of imports. If you don't want to use brackets, I can put everything on one line, but this will easily pass 120 characters, especially in the final version.

Reply:
Removed brackets, put all imports on a single line. The linter doesn't complain, so I won't either :)

Reply:
Now that I see that it is described by PEP 328, I like the idea. I think that we should use that going forward.